### ISSUE TITLE
[Linux x86_64] Massive FPS lag until switching window mode

### ISSUE THREAD

**Issue by keli5** [Facepunch/garrysmod-issues#5294]
### Details
On native Linux x86_64, whether starting in fullscreen or windowed, the fps is extremely low, about 30, when you load into the game, regardless of map. This is only resolved in my experience by switching the window mode, e.g. setting the game to windowed and then back to fullscreen, which then returns the game to 300+ FPS on my machine.
After some time of playing, the FPS will return to 30-45, and not even a map clean or reloading the map will fix it. At this point, you have to restart the whole game, since attempting to switch window mode now will likely lock up the game or completely crash it.
I'm fairly certain this isn't a hardware problem as I have a GT 1030 and an Intel Core i5-9500 with 16GB of RAM, definitely enough to handle the game on Windows as I did for months without this problem.

### Steps to reproduce
1. Open Garry's Mod on native Linux (x86_64 branch)
2. Load into any map
3. FPS is often 30 or less
4. Set game to Windowed and then back to fullscreen
5. FPS is back to normal

**Comment by keli5** [comment]
After reinstalling the game (same folder, same os, everything) the issue seems to be resolved?...

**Comment by seacat17** [comment]
This issue is still relevant.
